README.adoc

```diff
@@ -115,17 +92,22 @@ First step, you have to build the Docker image
 
 In order to run the build, you have to launch the following command:
 
-    $ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/spring/destination:/destination -t james/project -s SHA1
+    $ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -t james/project -s SHA1
```

Comment:
```
Multi-stage builds are a new feature in Docker 17.05
```

In my dev environment:

```
benwa@horizon ~/Documents/apache/onssd/james-project (git)-[JAMES-2048] % docker -v
Docker version 17.06.0-ce, build 3dfb8343
```

But on Yoda:

```
root@yoda:/home/btellier# docker -v
Docker version 1.11.2, build b9f10c9
```

Which means Docker upgrade on online servers....

Moreover, you are proposing a fancy new tech in a review about getting the job done. Maybe not the right place for this. I believe we should uncouple Java-8 migration from multi-stage builds. After all maybe this shiny proposal deserve a ticket by it's own, no?